CONTRIBUTING.md

```diff
@@ -79,6 +79,7 @@ optimize:
 When Kibana runs in development mode it will automatically use bundled SSL certificates. These certificates won't be trusted by your OS by default which will likely cause your browser to complain about the cert. You can deal with this in a few ways:
 
   1. Supply your own cert using the `config/kibana.dev.yml` file.
+    - You can point at the development certificate found in `test/dev_certs` 
```

Comment:
I'm confused. These bullet points are offering solutions to the problem where your OS doesn't trust the bundled dev certs, but this new bullet point is suggesting that you use the dev certs that Kibana already uses automatically. 

If the idea here is to highlight the fact that these dev certs are available, it should probably be done outside of these bullet points right?